assets/post_install_linux.sh

```diff
@@ -21,5 +21,8 @@ ${PREFIX}/bin/conda env config vars set PYTHONNOUSERSITE=1
 echo "ℹ️ Disabling mamba package manager banner."
 ${PREFIX}/bin/conda env config vars set MAMBA_NO_BANNER=1
 
+echo "ℹ️ Pinning BLAS implementation to OpenBLAS"
+echo "libblas=*=*openblas" >>${PREFIX}/conda-meta/pinned
```

Comment:
This prevents accidental switching of the BLAS implementation later on by the user when installing packages manually

Reply:
Nice!